docs/schemas/v1/definitions.json

```diff
@@ -47,6 +75,31 @@
                     ]
                   },
                   "description": "Available roles for this application. These can be assigned to users."
+                },
+                "users": {
```

Comment:
I think we can start with authentication only for this PR. 

I don't think it makes sense at the moment for the toolpad configuration to act as a user pool. This would require developers to have to hardcode each user in the configuration. And likely be a privacy problem for public repos.

Reply:
Ok, I can keep most of this PR on hold if that's the best course and see what can be merged for now.
We can also have a more in-depth discussion about this and see if there's any more solutions we can agree on.